### ISSUE TITLE
404 Error / Page Not Found page H1 needs to be tagged as a heading

### ISSUE THREAD

**Issue by eelzi-ONRR** [ONRR/nrrd#2344]
Right now the H1 on the 404 error page is tagged as text, not a heading. This is causing it fail to accessibility checkers and is prompting SEO issue reports in Bing Webmaster and Googls Search Console, since any failed redirect link points to the 404 page.

https://www.onrr.gov/404

**Comment by eelzi-ONRR** [comment]
Closing - this is a onrr.gov issue, not nrrd
